src/lib/util/statistics.ts

```diff
@@ -0,0 +1,390 @@
+import {
+  QuestionnaireByID,
+  ResponseResult,
+  QuestionDetails,
+  ResponseBody
+} from '/@/lib/apis'
+
+export const adjustQuestions = (
+  questionnaire: QuestionnaireByID,
+  results: ResponseResult[],
+  questions: QuestionDetails[]
```

Comment:
```suggestion
  questions: QuestionDetails[],
  results: ResponseResult[]
```
なんか順番が逆のほうが他と一貫性があっていい気がする